README.md

```diff
@@ -4,12 +4,20 @@
 
 Go to [README.md relative](./README.md)
 
+---
+
 Go to [README.md absolute](/README.md)
 
+---
+
 Go to [README.md basic](README.md)
```

Comment:
Can I get from here to [README.md](/README.md)

What if I use relative? [README.md relative](../../README.md)